scripts/release.sh

```diff
@@ -399,7 +399,7 @@ exec_helm() {
   if [ -n "$LOCAL_HELM" ]; then
     $HELM $@
   else
-    $NIX_SHELL -p "(import (import $NIX_SOURCES).nixpkgs {}).kubernetes-helm-wrapped" --run "$HELM $@"
+    $NIX_SHELL -p "(import (import $NIX_SOURCES).helm-nixpkgs {}).kubernetes-helm" --run "$HELM $@"
```

Comment:
Can't we just update the nixpkgs?

Reply:
That'd get us to 3.15.0, I was leaning more towards 3.15.4. WDYT?

Reply:
Alright, I guess 3.15.0 is just as good as 3.15.4. Closing this PR.

Reply:
But we still need to upgrade nixpkgs, or do we already have 3.15.0 here?